package.json

```diff
@@ -60,14 +60,14 @@
     "react-native-iap": "3.4.15",
     "react-native-image-crop-picker": "^0.25.2",
     "react-native-image-zoom-viewer": "^2.2.27",
-    "react-native-keyboard-aware-scroll-view": "^0.8.0",
+    "react-native-keyboard-aware-scroll-view": "^0.9.1",
     "react-native-linear-gradient": "^2.4.2",
     "react-native-modal-dropdown": "^0.7.0",
     "react-native-modal-popover": "0.0.12",
     "react-native-navigation-bar-color": "^0.1.0",
     "react-native-reanimated": "^1.3.0",
     "react-native-render-html": "^4.1.2",
-    "react-native-screens": "^2.0.0-alpha.16",
+    "react-native-screens": "^2.0.0-alpha.18",
```

Comment:
please revert to this version update. because i removed this library with #1394

Reply:
ok!